boot.go

```diff
@@ -106,9 +106,9 @@ func readSecrets() (string, string) {
 	return strings.TrimSpace(string(keyID)), strings.TrimSpace(string(accessKey))
 }
 
-func newMinioClient(host string, port int, accessKey, accessSecret string, insecure bool) (minio.CloudStorageClient, error) {
+func newMinioClient(host string, port string, accessKey, accessSecret string, insecure bool) (minio.CloudStorageClient, error) {
```

Comment:
@bacongobbler you can now write this as `host, port, accessKey, accessSecret string, insecure bool`